packages/frontend-shared/src/components/StandardModal.vue

```diff
@@ -77,7 +77,7 @@ const props = withDefaults(defineProps<{
 }>(), {
   modelValue: false,
   helpText: `${defaultMessages.links.needHelp}`,
-  helpLink: 'https://on.cypress.io',
+  helpLink: '',
```

Comment:
Any reason to make this default value `''`?

Reply:
Right I see - to hide the link for modals where a "Help Link" doesn't make sense.
Quick refactor to support this and to get CI green https://github.com/cypress-io/cypress/pull/23545

Reply:
Yeah, this just makes it so that if you don't pass a link then we don't show the link section in the modal header